SupportBank/Program.cs

```diff
@@ -1,12 +1,46 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.IO;
+using CsvHelper;
 
 namespace SupportBank
 {
     class Program
     {
+        private static Dictionary<string,Account> _accounts;
```

Comment:
You don't need the underscore prefix. Also is there any need to store this as a class variable? Static but not constant variables should generally be avoided as they can make the code harder to understand 🙂